### DISCUSSION TITLE
how do I run this on my G14?

### DISCUSSION THREAD

**Discussion by ctam19** [aredden/electron-G14Control#14] [General]
I am sure this is a newbie question.  I downloaded the files but I don't know how to run this program.

Can someone tell me how to do so?

Thanks :)

**Comment by aredden** [comment]
You would go here: https://github.com/aredden/electron-G14Control/releases/tag/v0.1.31 and then under assets, download the file called "G14ControlV2-Setup-0.1.31.exe"

**Comment by ctam19** [comment]
Thank you!
